src/rust/dbv2/migrations/2023-11-08-104312_volume_info/up.sql

```diff
@@ -0,0 +1,26 @@
+-- Your SQL goes here
+CREATE FUNCTION api.volume_info(market integer, "seconds" integer)
```

Comment:
How about consolidating these queries in a revision to the `small-queries` migration?

You can also include the nominal price change per:  `last_fill_price - first_fill_price`